pkg/cmd/util/output/backup_storage_location_printer.go

```diff
@@ -30,7 +30,7 @@ var (
 		{Name: "Name", Type: "string", Format: "name"},
 		{Name: "Provider"},
 		{Name: "Bucket/Prefix"},
-		{Name: "Status"},
+		{Name: "Phase"},
```

Comment:
I think `Status` is a better name.

Reply:
To clarify: I changed this (seemly unrelated to this PR) field to match the new column `Phase` we will be displaying with `kubectl`:

![image](https://user-images.githubusercontent.com/16508/85134582-97309680-b1f1-11ea-9e86-cf20fb8b00ea.png)

Pls ignore the 2 in `PHASE2`, I haven't refreshed my cluster.

The reason why I chose `phase` instead of `status` is because the field is `status.phase`, so this is more accurate and also leaves room to add other `status.x` fields if we want to.

Reply:
I like `Phase` here, since `Status` is the name of a field that contains other fields.

It also directly maps to the field's name.